cl/settings/10-public.py

```diff
@@ -579,3 +579,9 @@ def skip_unreadable_post(record):
     AWS_S3_CUSTOM_DOMAIN = "%s.s3.amazonaws.com" % AWS_STORAGE_BUCKET_NAME
 
 CLOUDFRONT_DOMAIN = ""
+
+####################################
+# Binary Transformers & Extractors #
+####################################
+
+BTE_URL = "http://cl-binary-transformers-and-extractors:80"
```

Comment:
I'm a little surprised you can run this on port 80, but I guess it works since it's inside the docker network. But I think we should pick and use a high-numbered port for this since the low ones are essentially spoken for. I'm no expert here, but can you think about it and look at port allocations and see which might make sense? Ideally, we just don't want it to be an existing port or one we may wind up at someday, and if it somehow makes sense, that's a benefit. Like 2020 might not be crazy, I dunno.

Reply:
2020, may never want to see that number again